data-tree/reanalysis/amsua/nasa/1998-2009.clean-bucket.yaml

```diff
@@ -0,0 +1,29 @@
+cycling_interval: 21600
+date_range:
+  datestr: '%Y%m%dT%H%M%SZ'
+  start: 19981001T000000Z
+  end:   20090501T000000Z 
+source:
+  -nasa-share:
+    name: "nasa share"
+    platform: aws_s3
+    path: "nasa/reanalysis/tovs/bufr/AMSUA_repros"
+    bucket: noaa-nasa-share
```

Comment:
is think this is the full path to the bucket
na-stage.us-east-2.noaa-nasa-share